web/src/app.js

```diff
@@ -247,3 +251,17 @@ konnektApp.controller('dashboardController', ['$scope', '$window', 'UserService'
 
   $scope.header = UserService.getUserData().email;
 }]);
+
+konnektApp.controller('editController', ['$scope', 'UserService', function ($scope, UserService) {
+
+  $scope.header = 'kontakt szerk';
+  $scope.welcome = 'Változtass az ismerőseiden!';
+  $scope.button = 'mehet';
```

Comment:
👍 with these, you might even be able to reuse this popup template for all the popups in the far future!